### PR TITLE
Update Firestore emulator to v1.18.2

### PR DESCRIPTION
**Pull request by HansiMou**
- Removed nano precision in timestamp in events.
- Fixed a bug where query behaves different from production.

### Description

- Removed nano precision in timestamp in events.
- Fixed a bug where query behaves different from production.

### Scenarios Tested

Unit tests.

### CODE REVIEW

**Comment by colerogers** [comment]
Can you add an entry to the CHANGELOG.md in the root of the project? 

Here's the format that they are in - https://github.com/firebase/firebase-tools/commit/cc2d7edc7e57dcc3ff739f6b2c2052598773253a#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

**Comment by HansiMou** [comment]
@colerogers added. Please check.

**Comment by colerogers** [comment]
@HansiMou Can you put those on one line since this is just one change going in?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- Removed nano precision in timestamp used in Firestore emulator. (#5893)
+- Fixed a bug where query behaves different from production.
```
Comment (colerogers) [inline CHANGELOG.md:2]: Combine these on one line

**Review by colerogers — APPROVED** [review]

**Review by colerogers — APPROVED** [review]

**Review by joehan — APPROVED** [review]
LGTM with a wording fix

**Pull request closed without merging**